CHANGELOG.md

```diff
@@ -13,6 +13,8 @@
 * Ref: `ITransport` implementations are now responsible for executing request in asynchronous or synchronous way (#1118)
 * Ref: Add option to set `TransportFactory` instead of `ITransport` on `SentryOptions` (#1124)
 * Ref: Simplify ITransport creation in ITransportFactory (#1135) 
+* Feat: Add non blocking Apache HttpClient 5 based Transport (#1136)
+* Enhancement: Autoconfigure Apache HttpClient 5 based Transport in Spring Boot integration (#1143)   
```

Comment:
this is part of the PR template next steps, but it's in the changelog as well, which one would be not up to date?